packages/blade/src/components/Dropdown/Dropdown.tsx

```diff
@@ -182,8 +184,13 @@ const _Dropdown = ({
   return (
     <BottomSheetAndDropdownGlueContext.Provider value={BottomSheetAndDropdownGlueContextValue}>
       <DropdownContext.Provider value={contextValue}>
-        <BaseBox position="relative" textAlign={'left' as never} {...getStyledProps(styledProps)}>
-          {children}
+        <BaseBox
```

Comment:
is wrapper needed on Dropdown?

Reply:
Yeah, without it Dropdown was breaking especially if you make it flex, the sub-components get messed up

Reply:
Ohh do we want to consider the flex scenarios as well? I was thinking more along the lines of that components shouldn't break when we pass `block` or `inline-block` to it. Because while doing responsiveness, those are the props that usually people use to "show" an element.

E.g.

```jsx
<Badge display={{ base: 'none', m: 'block' }}>I am badge visible on desktop</Badge>
```

Things breaking when we pass "flex" should be fine right? those will break the UI itself so it will be noticeable and won't be shipped. E.g. I can't think of scenario where someone would want to make Dropdown a display flex.

Reply:
I think we'll have to consider all scenarios right? It shouldn't break the component's visuals. Imagine someone shipped this code without testing on smaller screen and it breaks the component's UI itself. 

```tsx
<Badge display={{ base: 'none', s: 'flex' }}>I am badge visible on desktop</Badge>
```